blogs/{{How To Get Started: Coding}}/proposal.md

```diff
@@ -0,0 +1,25 @@
+📌 Step 1
+TITLE:
+{{How To Get Started: Coding}}
+
+TOPIC:
+{{Beginning to Code}}
+
+DESCRIPTION (5-7+ sentences):
+{{This blog's goal is to help readers who are interested in coding but don't quite know where to start. This blog will provide step by step instructions on setting up an GitHub account and navigating GitHub. This blog will help readers pick their ideal coding language and teach them how to start thinking like a coder. The blog will also offer detailed explanations and suggestions for a reader who wants to create their own website or make a game. It will also contain links to tutorials that will help the reader set up an API, website, and their own server! }}
```

Comment:
The scope of this blog is too broad. You need to narrow it down to either github, choosing a language, or getting started on a website or setting up APIs.